packages/react-components/react-datepicker-compat/src/index.ts

```diff
@@ -4,7 +4,12 @@ export type { CalendarNavigationIcons, CalendarProps, CalendarStyleProps, Calend
 export { CalendarDay, useCalendarDayStyles_unstable } from './CalendarDay';
 export type { CalendarDayProps, CalendarDayStyleProps, CalendarDayStyles, ICalendarDay } from './CalendarDay';
 
-export { CalendarDayGrid, useCalendarDayGridStyles_unstable } from './CalendarDayGrid';
+export {
+  calendarDayGridClassNames,
+  extraCalendarDayGridClassNames,
+  CalendarDayGrid,
+  useCalendarDayGridStyles_unstable,
```

Comment:
_nit:_ Alphabetically order these

```suggestion
  CalendarDayGrid,
  calendarDayGridClassNames,
  extraCalendarDayGridClassNames,
  useCalendarDayGridStyles_unstable,
```